scripts/release.sh

```diff
@@ -31,7 +31,11 @@ fi
 # Publish to Bintray. Will do nothing by default for safety unless overridden
 # via RELEASE_TRIGGER=true.
 if [ "$RELEASE_TRIGGER" = "true" ]; then
-    sbt release
+    # release Scala & then ScalaJS to avoid issue where release-with-defaults only releases Scala lib
+    # the next-version flag is to silence SBT's interactive release shell prompt - it doesn't actually alter the version
```

Comment:
This comment left over from previous version? (I don't see this flag here anymore)

Reply:
yes, left over! I'll move to `build.sbt` where it should be